<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QMainWindow" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblInstruction">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bring the two handles together and press the PLUTO Button</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblInstruction_2">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Calibration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCalibStatus">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPositionTitle">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Handle Distance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPositionDisplay">
         <property name="font">
          <font>
           <family>Bahnschrift Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblInstruction2">
       <property name="font">
        <font>
         <family>Bahnschrift Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
